LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Adfinis SyGroup AG
+Copyright (c) 2019 Adfinis SyGroup AG
```

Comment:
Isn't the convention to list all years with a release?

> Copyright (c) 2017, 2019 Adfinis SyGroup AG

https://www.gnu.org/licenses/gpl-howto.en.html (The copyright notice)

Reply:
Cool, didn't know that!

Reply:
fixed using `2017-2019`